lib/ClimaCoreVTK/test/runtests.jl

```diff
@@ -184,3 +184,31 @@ end
     )
     @test isfile(joinpath(OUTPUT_DIR, "hybrid3d_cell.vtu"))
 end
+
+@testset "hybrid 3d sphere" begin
+    R = 6.37122e6
+
+    hdomain = Domains.SphereDomain(R)
+    hmesh = Meshes.EquiangularCubedSphere(hdomain, 4)
+    htopology = Topologies.Topology2D(hmesh)
+    quad = Spaces.Quadratures.GLL{5}()
+    hspace = Spaces.SpectralElementSpace2D(htopology, quad)
+
+
+    vdomain = Domains.IntervalDomain(
+        Geometry.ZPoint(0),
+        Geometry.ZPoint(20.0),
+        boundary_names = (:bottom, :top),
+    )
+    vmesh = Meshes.IntervalMesh(vdomain, nelems = 20)
+    vtopology = Topologies.IntervalTopology(vmesh)
+    vspace = Spaces.FaceFiniteDifferenceSpace(vtopology)
+
+    fspace = Spaces.ExtrudedFiniteDifferenceSpace(hspace, vspace)
+    cspace = Spaces.CenterExtrudedFiniteDifferenceSpace(fspace)
+    writevtk(
+        joinpath(OUTPUT_DIR, "hybrid_sphere_point"),
+        Fields.coordinate_field(fspace);
```

Comment:
Is this supposed to show all points in this `fdspace` in the z direction? (i.e., showing the thickness from 0 to 20?) 
I tested this but I only see the 2D manifold
![vtk-test-hybrid_sphere_point-output](https://user-images.githubusercontent.com/39932030/149427841-501c5c4b-8057-4f9e-8212-b7b8e2181ae7.png)

Reply:
I think because it's really thin?

If you add an isosurface for `z` it gives you the different levels

Reply:
30 km vs 6400 km is a big difference

Reply:
That's fair.. I was also zooming in and I could see through :eyes: 
I'll try a different value

Reply:
I wonder if for these viz tests would make sense to go dimensionless instead of dimensional ...

Reply:
@OsKnoth suggested an option to exaggerate the z coordinate.

Reply:
I feel that would be better to have as a filter-pipeline in paraview (post-process)